flake.nix

```diff
@@ -34,7 +35,30 @@
             cabal-fmt
             fourmolu;
         };
-        packages.default = config.packages.mood-tracker;
```

Comment:
why remove this?

Reply:
bonus question: can you explain what this line does?

Reply:
This has since been added back in. As to the second issue, this is the default package that Nix is to build/run.

As explained in the deploy-rs docs 
```
    # This is the application we actually want to run
    defaultPackage.x86_64-linux = import ./hello.nix nixpkgs;
```

In general to help with all of this I will and am dedicating some time to the NixOS docs again.

Reply:
That doesn't explain what `packages.default = config.packages.mood-tracker;` does. You can learn the Nix language a bit thoroughly to be able to answer questions like this. It is more to do with the language, rather than NixOS.

Reply:
I will I'll go over things again, yep.

Reply:
Dot Notation is used to access elements of an attribute-set, thus here you are setting the value of the default attribute of the attribute-set known as "packages". To be the value contained within the attribute packages, itself contained within the attribute-set known as "config".

This is my present understanding.

Reply:
👍🏿